src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const errorHandler = (err) => {
+  const divTag = document.createElement('div');
+
+  divTag.classList.add('message', 'error-message');
+  divTag.textContent = `${err.message}`;
+
+  document.body.append(divTag);
+};
+
+const successHandler = () => {
+  const divTag = document.createElement('div');
+
+  divTag.classList.add('message');
+  divTag.textContent = 'Promise was resolved!';
+
+  document.body.append(divTag);
+};
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    resolve();
+  });
+});
+
+promise1.then(successHandler);
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Promise was rejected!'));
```

Comment:
You can also move it to constant

Reply:
@DanilWeda you mean 'Promise was rejected!'?))))